src/components/Table/Table.tsx

```diff
@@ -173,19 +166,25 @@ function Table<T>({
   const handleSorting = (column: string) => {
     setSorting(prevState => {
       if (sortDir) {
+        onSort?.(column, sortDir);
+
         return {
           column,
           order: sortDir,
         };
       }
 
       if (prevState.column !== column) {
+        onSort?.(column, 'asc');
+
         return {
           column,
           order: 'asc',
         };
       }
 
+      onSort?.(column, prevState.order === 'asc' ? 'desc' : 'asc');
```

Comment:
`onSort` is called twice? 

1) Will be called on both line 169 or 178 
2) line 186

Why is that?